Add slidedeck post (#136)
dapr/blog: daprblog/content/posts/2024/new-dapr-slidedeck/index.md

---
date: "2024-03-10T07:00:00-07:00"
title: "New Dapr slide deck for your presentations"
linkTitle: "New Dapr slide deck for your presentations"
author: Marc Duiker
type: blog
---

A new [Dapr slide deck](https://docs.dapr.io/contributing/presentations/) is available that you can use for your Dapr presentations!

![Slide deck overview](slide-overview.png)

> The deck contains new icons for all Dapr APIs and built-in features such as security, resiliency, and observability. The presentation style matches the [dapr.io](http://dapr.io) website that was redesigned a while ago.

Reviewer: ```suggestion
The deck contains new icons for all the Dapr APIs and built-in features, such as security, resiliency, and observability. The presentation style matches the recently redesigned [dapr.io](http://dapr.io) website.
```

Here are some slides from the deck:

![From Development to Hosting](slide17.png "From Development to Hosting")

![Components](slide18.png "Components")

![Using the Dapr APIs](slide20.png "Using the Dapr APIs")

![Workflow example](slide52.png "Workflow example")

> The slide deck is large (over 140 slides) and is intended to be used as a starting point for your presentation. Remove or hide what is not necessary for your audience, and add new slides to add relevance to your story.

Reviewer: ```suggestion
The slide deck consists of over 140 slides that can be used as a starting point for your presentation. Remove or hide what is not necessary for your audience, and add new slides to add relevance to your story.
```

Reviewer: I think this paragraph might fit better as part of line 15... wdyt?

> There are instructions on the first 3 slides on building new slides and diagrams. The last 20 slides in the deck contain visual assets you can use.

Reviewer: ```suggestion
Follow the instructions on the first 3 slides to get started on building new slides and diagrams. The last 20 slides in the deck contain visual assets you can use.
```

![Visual assets](asset-slides.png "Visual assets")

> The [Dapr slide deck](https://docs.dapr.io/contributing/presentations/) (PowerPoint format) is open source, so contributions are welcome!

Reviewer: ```suggestion
Like the Dapr diagram slide deck, the [Dapr presentation slide deck](https://docs.dapr.io/contributing/presentations/) (PowerPoint format) is open source, so contributions are welcome!
```
